### PR TITLE
Our givit-app now can run by one command

### PR DESCRIPTION
**Pull request by rotem-benzvi**
The commit:
1. bootstrap.sh now initial and run the server automate
2. for sync all python-lib we need pipfile's
3. we dont need requiement.txt anymore since we change the aproach

fixed:#20

### CODE REVIEW

File: bootstrap.sh [hunk 1]
```diff
@@ -2,11 +2,19 @@
...
-apt install -y python3.7 python3-pip postgresql postgresql-contrib
-sudo apt install libpq-dev
+apt install -y python3.7 python3.7-dev python3-pip postgresql postgresql-contrib libpq-dev postgresql-common postgresql-client #postgresql-devel
 pip3 install --upgrade pip
-pip3 install -r /vagrant/requirements.txt
+
+# Installing pipenv, create new env and sync lib with pipfile
+pip3 install pipenv
+cd /vagrant/givitsite/
+pipenv run *
```
Comment (ifireball) [inline bootstrap.sh:11]: not sure what are you trying to do here, run all the files in this directory before even running `sync` ?

**Review by ifireball — COMMENTED** [review]
there is one thing there that isn't clear to me

File: bootstrap.sh (hunk 1 above)
Comment (rotem-benzvi) [reply bootstrap.sh]: you right, i thought that this initial the ens but its not, anyway its work well without it (check), and erase this line and commit it bellow..

**Review by ifireball — APPROVED** [review]
LGTM

**Pull request merged**